src/services/DataTools.js

```diff
@@ -0,0 +1,72 @@
+const ColorConsole = require('../services/ColorConsole');
+// TODO use getTestRailStatusSkipped, getTestRailStatusFailed, and getTestRailStatusPassed Instead of it 2, 5, 1
+
+class DataTools {
```

Comment:
I want to use the object as potential library. If we do tackle parsing I'd make it a method that I would add here so I'd like to have a library style name and not one bound to that it's one and only method..

Reply:
it's a class with a 50 line method, esp as a class you should not be stuffing completely unrelated things in it. 

It doesn't even really need to be a class but that's a separate topic.

If you want a shared.js or utils.js or misc.js to stuff a bunch of random helper functions they should generally be so small that they don't warrant their own module (5-10 line functions, and still somewhat related in purpose.)

If we were going to do other things related to processing or transforming results prior to uploading them to TR, that would make sense to put in this module.  Then you could call it something like ResultsProcessor.

Reply:
First and very much related to the notion of library, this but if you want to export some thing, it has to be an object.     

I tried exporting the method originally as a function, and it's not allowed in a module the only way to use it is to use the formulation.  module.exports = Obj.

 Why not treat this as a library from the gecko rather than demand that since there's only one method that it has to have I related name even though I know that I want to add to it. It's more flexible as a library anyway.

Reply:
You can very much export a function.  Try

```
function aggregateTestResults(cytrPostData) {
  // blabhabla
}

module.exports = {
  aggregateTestResults,
};
```

Reply:
When I tried this, I got an error. When I actually round the code, said that export was not supported with modules. Have you actually run the code like thi

Reply:
oh like that

Reply:
Given that I'd like to drop the use of an object call the file datatools.js

And export functions on a one off basis as you outlined.